Fabric.Identity.API/appsettings.json

```diff
@@ -42,6 +43,7 @@
   "ExternalIdProviderSettings": {
     "ExternalIdProviders": []
   },
+  "AzureActiveDirectorySettings": {},
```

Comment:
Do we want to add some default settings here, so we can add comments for what the settings are for?

Reply:
done